spark/src/main/scala/geotrellis/spark/reproject/Reproject.scala

```diff
@@ -0,0 +1,23 @@
+package geotrellis.spark.reproject
+
+import geotrellis.raster.reproject.Reproject.{Options => RasterReprojectOptions}
```

Comment:
Not psyched about this. I don't the name clashing that is going on with `geotrellis.vector.reproject.Reproject`, `geotrellis.raster.reproject.Reproject`, and now this Reproject. Need to think of a better pattern here.